docker/nginx/conf.d.templates/server.dnslink.conf

```diff
@@ -12,5 +12,14 @@ server {
     ssl_certificate     /etc/ssl/local-certificate.crt;
     ssl_certificate_key /etc/ssl/local-certificate.key;
 
+    set_by_lua_block $skynet_portal_domain { return "{{PORTAL_DOMAIN}}" }
+    set_by_lua_block $skynet_server_domain {
+        -- fall back to portal domain if server domain is not defined
```

Comment:
why do we want this fallback, seems like it just obfuscates the problem a bunch

Reply:
that's because we have a requirement that SERVER_DOMAIN should be empty if it's one-server setup, in this case PORTAL_DOMAIN is the SERVER_DOMAIN